app/helpers/ops_helper/textual_summary.rb

```diff
@@ -125,7 +125,7 @@ def textual_vmdb_tables_most_wasted_space
       :title     => _("Tables with Most Wasted Space"),
```

Comment:
@europ move the constant definition on the beginning of this method, instead of leaving a random number in the code